src/settings/Settings.tsx

```diff
@@ -336,8 +337,23 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         isFeature: true,
         labsGroup: LabGroup.Rooms,
         supportedLevels: LEVELS_FEATURE,
-        displayName: _td("New spotlight search experience"),
-        default: false,
+        displayName: _td("New search experience"),
+        default: false,
+        betaInfo: {
+            title: _td("The new search beta"),
+            caption: () => <>
+                <p>{ _t("A new, quick way to search spaces and rooms you're in.") }</p>
+                <p>{ _t("This feature is a work in progress, we'd love to hear your feedback.") }</p>
+            </>,
+            disclaimer: () => <>
+                <h4>{ _t("How can I give feedback?") }</h4>
+                <p>{ _t("To feedback, join the beta, start a search and click on feedback. Or alternatively, click here.") }</p>
```

Comment:
Should this contain a link of some sort ? the copy says "click here" ?

Reply:
Yup, tweak incoming (copy and some other bits)